src/manifold_dual_contouring.rs

```diff
@@ -391,18 +395,20 @@ fn subsample_octtree<S: RealField + Float + From<f32>>(base: &[Vertex<S>]) -> Ve
 }
 
 struct Timer {
-    t: ::time::Tm,
+    t: std::time::Instant,
 }
 
 impl Timer {
     fn new() -> Timer {
-        Timer { t: ::time::now() }
+        Timer {
+            t: std::time::Instant::now(),
+        }
     }
-    fn elapsed(&mut self) -> ::time::Duration {
-        let now = ::time::now();
+    fn elapsed_millis(&mut self) -> u128 {
```

Comment:
can we change this to:
fn elapsed(&mut self) -> std::time::Duration 
?

Reply:
you have to do .to_millis() or whatever at every callside but I agree that the return type would read better, idk

Reply:
I think we could just use the Debug trait of Duration, and change the println!(".... {:}...", t.elapsed()); to
println!(".... {:?}...", t.elapsed());

Reply:
sure that works as well